leaderelection/config.go

```diff
@@ -30,6 +30,10 @@ import (
 
 const ConfigMapNameEnv = "CONFIG_LEADERELECTION_NAME"
 
+// MaxBuckets is the maximum number of buckets to allow users to define.
+// This is a variable so that it may be customized in the binary entrypoint.
+var MaxBuckets uint32 = 10
```

Comment:
Also why uint32: here and in the CM.
wouldn't int be just as good?  We validate the value anyway (against maxbuckets, so can just as well check > 0). And this would allow us not to add one more method to the cm parsing library.

or `for i := uint32(0); ` 🌮

Reply:
> this would allow us not to add one more method to the cm parsing library

I always get a little annoyed with our helpers (e.g. `ptr`) when certain basic types are missing, and my choice is either patching upstream (heavy) or casting downstream.

It's less annoying here because I'm not calling an API that requires `uint32`, but it's also easy enough to add the helper while I'm here 😇